### PR TITLE
✨ add dev config for local development

### PR DESCRIPTION
**Pull request by munroTom**
Set up a firebase project to use when deving so we don't need to use the live databases etc

A lot of it is prettier changes, let me know if you want me to go back and undo them, I've had a click round the app and it doesn't seem to have caused any issues

### CODE REVIEW

File: src/components/EmailVerifiedDialog.js [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import CloseIcon from "@material-ui/icons/Close";
+import withMobileDialog from "@material-ui/core/withMobileDialog";
+import { withStyles } from "@material-ui/core/styles";
+
+import authFirebase from "features/firebase/authFirebase";
```
Comment (munroTom) [inline src/components/EmailVerifiedDialog.js]: Just updated imports here and reordered

File: src/components/FeedbackReports/FeedbackDetailsPage.js [hunk 2]
```diff
@@ -1,59 +1,72 @@
...
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import withMobileDialog from "@material-ui/core/withMobileDialog";
+import { withStyles } from "@material-ui/core/styles";
+import CircularProgress from "@material-ui/core/CircularProgress";
 
-import { isIphoneWithNotchAndCordova, isIphoneAndCordova } from '../../utils';
-import config from '../../custom/config';
-import dbFirebase from '../../dbFirebase';
+import { dbFirebase } from "features/firebase";
```
Comment (munroTom) [inline src/components/FeedbackReports/FeedbackDetailsPage.js]: Just imports here

File: src/components/FeedbackReports/FeedbackReportsPage.js [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
+import Divider from "@material-ui/core/Divider";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import { withStyles } from "@material-ui/core/styles";
 
-import PageWrapper from '../PageWrapper';
-import dbFirebase from '../../dbFirebase';
-import config from '../../custom/config';
-import '../ModeratorPage.scss';
+import { dbFirebase } from "features/firebase";
```
Comment (munroTom) [inline src/components/FeedbackReports/FeedbackReportsPage.js]: Imports

File: src/components/LoginFirebase.js [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+import withMobileDialog from "@material-ui/core/withMobileDialog";
 
-import * as firebaseui from 'firebaseui';
-import firebase from 'firebase/app';
-import 'firebase/auth';
+import * as firebaseui from "firebaseui";
+import firebase from "firebase/app";
+import "firebase/auth";
+
+import { authFirebase } from "features/firebase";
```
Comment (munroTom) [inline src/components/LoginFirebase.js]: imports

File: src/components/WriteFeedbackPage.js [hunk 5]
```diff
@@ -1,73 +1,74 @@
...
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import TextField from "@material-ui/core/TextField";
+import { withStyles } from "@material-ui/core/styles";
+
+import { dbFirebase } from "features/firebase";
```
Comment (munroTom) [inline src/components/WriteFeedbackPage.js]: imports

**Comment by sebastianovide** [comment]
@munroTom  conflicts

**Comment by munroTom** [comment]
@sebastianovide have fixed the conflicts

**Pull request merged**